.shippable.yml

```diff
@@ -75,6 +75,7 @@ build:
     - _make PLATFORM=ls-ls1043ardb CFG_ARM64_core=y
     - _make PLATFORM=ls-ls1046ardb CFG_ARM64_core=y
     - _make PLATFORM=ls-ls1012ardb CFG_ARM64_core=y
+    - _make PLATFORM=ls-ls1012afrwy CFG_ARM64_core=y
```

Comment:
`make PLATFORM=ls-ls1012afrwy` (without `CFG_ARM64_core=y`) fails to build, and same for the two other flavors added in this PR.
I think, either the error(s) should be fixed, or `CFG_ARM64_core=y` should be forced in `plat-ls/conf.mk`.